app/javascript/components/applications/application_form/preferences/PreferencesSection.js

```diff
@@ -49,7 +49,7 @@ const PreferencesSection = ({ formApi, listingPreferences, editValues }) => {
       { shortFormPrefValues && shortFormPrefValues.map( ( pref, i ) => (
```

Comment:
Can't comment that far away from the code, but on line https://github.com/Exygy/sf-dahlia-lap/pull/61/files#diff-5a9037caa5d9887890817c4863ed8bf8R42 above, we're filtering out all of the households. It looks like it's because DOB isn't defined. Is that a known issue @chrisdolendo?

Reply:
this is related to the bug that maybe wasn't a bug on the bug branch, so please ignore